.trivyignore

```diff
@@ -10,6 +10,7 @@ CVE-2023-25153
 CVE-2023-25173
 CVE-2022-3821
 CVE-2022-41723
+CVE-2023-27561
```

Comment:
looks like this new high vulnerability in telegraf. do you know if this has been addressed latest version of the telegraf?

Reply:
> looks like this new high vulnerability in telegraf. do you know if this has been addressed in latest version of the telegraf?

I see this has been adressed in telegraf 1.26.1 - https://github.com/influxdata/telegraf/blob/master/CHANGELOG.md#v1261-2023-04-03.  @jatakiajanvi12 , do you know if the Mariner v2 has this version available? if not, we should create dependency ask for the Mariner team.

Reply:
@jatakiajanvi12  will be creating task for the Mariner team to get the telegraf 1.26.1 available which would address this vulns.